server/imagefit.py

```
@@ -0,0 +1,67 @@
+"""האם אימג' נכנס לכונן יעד, ואם לא — מה צריך לכווץ (‏#87).
+
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)